pcap/defs_windows_386.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by cmd/cgo -godefs; DO NOT EDIT.
```

Comment:
is it possible to add copyright notices to the godef files?  if not, no worries.

Reply:
not directly (cgo removes all comments :( )
But I reworked code generation to not use a bat file (now the whole process is more extensible) - now the generated ones have copyright notices